### PR TITLE
Only minor changes

### PR DESCRIPTION
**Pull request by xyproto**
* 50 fewer lines
* introducing the `must` function and `defer` in `main.go`
* favoring the "early return" style, which allows for reduction of indentation levels
* avoiding named returns in a couple of places (but there is not much of a difference for these particular cases)

Accept or reject at will.

Thanks for creating yay!

### CODE REVIEW

File: cmd.go [hunk 1]
```diff
@@ -373,35 +367,33 @@ func handlePrint() (err error) {
...
-func handleYogurt() (err error) {
+func handleYogurt() error {
 	config.SearchMode = NumberMenu
-	err = numberMenu(cmdArgs.targets)
-
-	return
+	return numberMenu(cmdArgs.targets)
 }
 
 func handleSync() (err error) {
```
Comment (Jguer) [inline cmd.go]: If these changes are made, it is not needed to name the return

Comment (xyproto) [reply cmd.go]: Agree, I can update the pull request.

**Comment by xyproto** [comment]
The desired granularity for pull requests differ from project to project, and is hard to specify in the contribution guidelines.

**Comment by xyproto** [comment]
Split out the changes to `main.go` to #602.

**Comment by Morganamilo** [comment]
As long as this is done consistently I don't mind. Might wanna rebase this against #585 though, as that's gonna be the next thing that gets merged as soon a we get a release.

**Comment by xyproto** [comment]
Thanks for merging!

**Pull request merged**